ompi/mca/coll/tuned/coll_tuned_decision_fixed.c

```diff
@@ -644,7 +652,12 @@ int ompi_coll_tuned_allgatherv_intra_dec_fixed(const void *sbuf, int scount,
     }
 
     /* Determine complete data size */
-    ompi_datatype_type_size(sdtype, &dsize);
```

Comment:
This looks good, I verified that ompi_coll_base_allgatherv_intra_two_procs already has a check for IN_PLACE, so no need to add above that.  Looks good.